Untitled.ipynb

```diff
@@ -37,6 +37,59 @@
     "a+b -1"
```

Comment:
<p>Good you left it untouched!</p>

 _Reply via <a href='https://app.reviewnb.com/slomeks/test-reviewnb/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>